package.xml

```diff
@@ -40,6 +40,7 @@
   
   <exec_depend>rclcpp</exec_depend>
   <exec_depend>rmw_implementation</exec_depend>
+  <exec_depend>launch_ros</exec_depend>
```

Comment:
`test_depend` perhaps?

Reply:
I tried that, it doesn't compile with that even with launch_ros installed. I consulted @clalancette and he said launch_ros needs to be exec_depend here. It compiles with that for both of us. Should we give that a try?

Reply:
Ah, no I think you're right. I was a little quick on the trigger there.